frontend/app/src/components/inputs/CounterpartyInput.vue

```diff
@@ -0,0 +1,37 @@
+<script lang="ts" setup>
+const props = withDefaults(
+  defineProps<{
+    value?: string | null;
+  }>(),
+  {
+    value: null,
+  },
+);
+
+const emit = defineEmits<{
+  (e: 'input', value: string): void;
+}>();
+
+const vModel = useSimpleVModel(props, emit);
+
+const rootAttrs = useAttrs();
+
+const { counterparties }
+  = useHistoryEventMappings();
```

Comment:
We should be able to change this to a single line right?

Reply:
I dont know why the auto formatting not working in webstorm 😫